lib/editor/narrative_editor.dart

```diff
@@ -19,7 +19,7 @@ class NarrativeEditorState extends State<NarrativeEditor> {
   var interactive = false;
 
   void openNode(StoryEditorState editor, Node node) async {
-    await Navigator.push(
+    await Navigator.push<Null>(
```

Comment:
The [`prefer-void-to-null`](https://dart-lang.github.io/linter/lints/prefer_void_to_null.html) rule complains about this (and all other uses of `Null`)